mmv1/third_party/terraform/services/storagetransfer/resource_storage_transfer_job.go.tmpl

```diff
@@ -695,14 +696,25 @@ func gcsDataSchema() *schema.Resource {
 			},
 			"path": {
 				Optional:    true,
-				Computed:    true,
 				Type:        schema.TypeString,
 				Description: `Google Cloud Storage path in bucket to transfer`,
+				ValidateFunc: validatePath,
 			},
 		},
 	}
 }
 
+func validatePath(v interface{}, k string) (ws []string, errors []error) {
```

Comment:
can this name be more specific to this resource. The namespace is shared with other resources in the service